src/main/resources/db/migration/V25__iat_ddl.sql

```diff
@@ -0,0 +1,2 @@
+-- Updating user emails to always be lowercase
+update tims_user set email = LOWER(email);
```

Comment:
User emails will always be lowercased moving forward.  Need to fix existing users.